specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/virtualMachine.json

```diff
@@ -1891,6 +1891,72 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/attachDetachDataDisks": {
+      "post": {
+        "tags": [
+          "VirtualMachines"
+        ],
+        "operationId": "VirtualMachines_AttachDetachDataDisks",
+        "description": "Attach and detach data disks to/from the virtual machine.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
```

Comment:
use the latest "v5" common type:

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/parameter-not-using-common-types.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/parameter-not-using-common-types.md)

Reply:
@ms-zhenhua  Currently in Microsoft.Compute I cannot see the version v5 specified anywher . I can see even in main we are in v1 version. Can you please elaborate how v5 can be specified. Not clear with the ask here